js/qz-tray.js

```diff
@@ -615,7 +615,12 @@ var qz = (function() {
                     //disable secure ports if page is not secure
                     if (typeof location === 'undefined' || location.protocol !== 'https:') {
                         if (options == undefined) { options = {}; }
-                        options.usingSecure = false;
+
+                        //respect forcing secure ports if it is defined, otherwise disable
+                        if (options.usingSecure === 'undefined') {
```

Comment:
Interesting, pretty sure I grabbed the exact code I was running that was working fine, but I've also been burning the midnight oil and JavaScript isn't my language of choice, so adjust as needed. 👍